removals/mp_gain_year_count_all_forest_types.py

```diff
@@ -142,7 +142,7 @@ def mp_gain_year_count_all_forest_types(tile_id_list):
 
         # Creates gain year count tiles using only pixels that had neither loss nor gain pixels
         if cn.count == 96:
-            processes = 90   # 66 = 360 GB peak; 88 = 430 GB peak; 90 = 510 GB peak
+            processes = 50   # 66 = 360 GB peak; 88 = 430 GB peak; 90 = 510 GB peak
```

Comment:
Replace the comment with 50 = XXX GB peak.

Reply:
Changes made in the following commit: bcf63c1c802f8e9327579dfffe90b2654b52ff53